feat(changelog): interlink-changed-scaleway-interlink-is-now-in-general- 2025-10-02 (#5607)
scaleway/docs-content: changelog/october2025/2025-10-02-interlink-changed-scaleway-interlink-is-now-in-general-.mdx

---
title: Scaleway InterLink is now in General Availability
status: changed
date: 2025-10-02
category: network
product: interlink
---

> Scaleway InterLink in now available in General Availability for both flavors:

Reviewer: ```suggestion
Scaleway InterLink is now in General Availability for both InterLink types:
```

>  - InterLink Hosted: connect you on-premise infrastructure to Scaleway VPC thanks to a mutualized third party interconnection

Reviewer: ```suggestion
 - InterLink Hosted: connect your on-premises infrastructure to your Scaleway VPC via a partner, using their mutualized third-party interconnection
```

>  - InterLink Hosted: connect your on-premises infrastructure to your Scaleway VPC via a partner, using their mutualized third-party interconnection

Reviewer: ```suggestion
 - InterLink Hosted: connect your on-premises infrastructure to your Scaleway VPC via a partner, with a mutualized third-party interconnection
```

>  - InterLink Self-Hosted: connect directly your on-premise infrastructure to Scaleway VPC with a physical dedicated connection

Reviewer: ```suggestion
 - InterLink Self-Hosted: directly connect your on-premises infrastructure to your Scaleway VPC with your own physical dedicated connection
```
